.github/workflows/build-and-test.yaml

```diff
@@ -44,11 +44,6 @@ jobs:
     if: ${{ !github.event.pull_request.draft }}
     runs-on: ubuntu-latest
     steps:
-      - name: Install protobuf
```

Comment:
I think it would be a good to have a job building with `generate-protos` enabled

Reply:
I agree, I think it would be a good idea having the build job building the proto files so it checks whether it's the correct submodule version that is pinned.

Reply:
Well as discussed on slack, the main purpose of this is that we don't regenerate proto files every time we update the dependency. So this is explicitly not possible.

Concretely we are not exposing the in-progress protobuf definitions until node version 6 is stable.